atom/common/platform_util.h

```diff
@@ -26,7 +26,11 @@ void OpenItem(const base::FilePath& full_path);
 void OpenExternal(const GURL& url);
 
 // Move a file to trash.
-void MoveItemToTrash(const base::FilePath& full_path);
+#if defined(OS_WIN)
```

Comment:
We should also return `bool` on Linux and OS X otherwise the API will return different types on different platforms.

And can you also add documents on `moveItemToTrash`'s new behavior?